package-system/AWSNativeSDK/Dockerfile

```diff
@@ -26,20 +45,17 @@ RUN DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata locales keyboard-
 RUN apt-get install -y cmake \
                        clang-12 \
                        ninja-build \
-                       libssl-dev \
-                       libssh-dev \
-                       zlib1g-dev \
-                       libcurl4-openssl-dev
+                       git \
+                       build-essential \
+                       autoconf \
+                       libtool
+
 
+RUN mkdir -p $WORKSPACE/curl && \
+    git -C $WORKSPACE/curl clone https://github.com/curl/curl.git --single-branch --recursive --branch curl-7_68_0 src
```

Comment:
We should probably store the `curl-7_68_0 src` branch in a variable, if for some reason the AWS SDK for C++ updates to a new version of curl in the future.